packages/headings/LinkedHeading.jsx

```diff
@@ -37,15 +50,24 @@ export function LinkedHeading({
           data-test="cta"
           href={ctaLink}
           target={ctaNewWindow ? "_blank" : undefined}
-          className="su-flex su-no-underline hover:su-underline hover:su-text-digital-red dark:hover:su-text-dark-mode-red su-transition su-items-center md:su-items-end md:su-mb-8 lg:su-mb-12 su-text-black dark:su-text-white su-flex-nowrap su-gap-20 md:su-gap-13 su-align-baseline su-text-19"
+          className={cnb(
+            "su-group su-flex su-no-underline hocus:su-underline su-transition su-items-center md:su-items-end md:su-mb-8 lg:su-mb-12 su-flex-nowrap su-align-baseline su-gap-20 md:su-gap-13 su-text-19 su-decoration-2 dark:su-text-white",
+            isAlwaysLight
+              ? "su-text-white hocus:su-text-white/95 dark:hocus:su-text-white/95"
```

Comment:
I use @broleomo 's trick here. Siteimprove wants a color change, but a color change to dark mode red makes it less accessible, so I'm using a hocus color of 95% white which counts as a color change but not noticeable to the eyes 😆